crates/agent/entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# Place secrets into expected file locations.
+mkdir /root/.aws
```

Comment:
We are not placing anything here, is this actually necessary to create here manually?

Reply:
I left it because it's likely we will in the future, if we have agent APIs doing s3 operations (such as for updating storage mappings).